teamC/sungsik/routes/likesRouter.js

```diff
@@ -0,0 +1,12 @@
+const express = require('express')
+const asyncWrap = require('../errorHandler/asyncWrap')
```

Comment:
아래와 같이 import 하실 때에 객체를 풀어서 선언하시면 아래 코드가 간결해 질 것 같습니다.

```suggestion
const { asyncWrap } = require('../errorHandler/asyncWrap')
```